content/authentication/reading/getting-starting-identity/_index.md

```diff
@@ -0,0 +1,391 @@
+                .WithMany(b => b.events);
+
+            modelBuilder.Entity<Event>()
+                .HasMany(p => p.Tags)
+                .WithMany(p => p.Events)
+                .UsingEntity(j => j.ToTable("EventTags"));
+            base.OnModelCreating(modelBuilder);
+        }
+    }
+}
+```
+
+You may note that we didn't add any `DbSet` for `IdentityUser` like we did for other models in the application. This is not an oversight! With `EventDbContext` properly set up, we can run a migration and the database will add the appropriate tables for our authentication data.
+
+Add the following line to your `Program.cs` file:
+
+```csharp
+builder.Services.AddRazorPages();
+```
+
+{{% notice blue "Note" "rocket" %}}
+While you are editing your `Program.cs` file, you may need to also add a default identity user. We will learn more about how to configure this user in a later section. For now, you can add code to `Program.cs` to address this:
+{{% /notice %}}
+
+```csharp {linenos=table}
+builder.Services.AddDefaultIdentity<IdentityUser>
+(options =>
+{
+   options.SignIn.RequireConfirmedAccount = true;
+   options.Password.RequireDigit = false;
+   options.Password.RequiredLength = 10;
+   options.Password.RequireNonAlphanumeric = false;
+   options.Password.RequireUppercase = true;
+   options.Password.RequireLowercase = false;
+}).AddEntityFrameworkStores<EventDbContext>();
+```
+
+You will also need to add a couple lines of code below `app.UseAuthorization();`
+
+```csharp
+app.MapRazorPages();
+app.MapControllers();
+```
+
+`app.MapRazorPages();` specifies to the app that the Identity pages should follow the routing laid out in `_LoginPartial.cshtml`.
+
+These initial steps were to make sure that the application is still using `EventDbContext` for its connection to the database now that we have added Identity. However, if you take a look inside the `Areas/Identity/Data` directory, you will find a file also called `EventDbContext`. Delete that generated file and continue to use the one we initially created for `CodingEvents`.
+
+You will also need to remove the following line from your `Program.cs` file:
+
+```csharp
+using CodingEvents.Areas.Identity.Data;
+```
+
+The CodingEvents application already has a connection string in addition to using our builder to add database context configured. Since we also configured our own default `identity` user earlier in the walkthrough we can also safely remove that as well. You will need to remove the following lines of code from your `Program.cs` file that were generated when adding our identity scaffolding:
+
+```csharp {linenos=table}
+var connectionString = builder.Configuration.GetConnectionString("EventDbContextConnection");builder.Services.AddDbContext<EventDbContext>(options =>
+    options.UseSqlServer(connectionString));builder.Services.AddDefaultIdentity<IdentityUser>(options => options.SignIn.RequireConfirmedAccount = true)
+    .AddEntityFrameworkStores<EventDbContext>();
+```
```

Comment:
This code blocks formatting is off ->

<img width="994" alt="codeBlockFormatting" src="https://user-images.githubusercontent.com/39868164/225022785-6b3d5d14-4dec-42d3-9600-6ac738253f11.png">

Reply:
resolved this issue during pairing.